src/ProjectService.Models.Dto/Requests/Filters/FindTasksFilter.cs

```diff
@@ -0,0 +1,17 @@
+﻿using Microsoft.AspNetCore.Mvc;
+using System;
+
+namespace LT.DigitalOffice.ProjectService.Models.Dto.Requests.Filters
+{
+    public class FindTasksFilter
+    {
+        [FromQuery(Name = "number")]
+        public int? Number { get; set; }
+
+        [FromQuery(Name = "projectId")]
```

Comment:
As i know, we are using low-case in filters according to filter in UserService

Reply:
projectid, assignto

Reply:
Fixed